<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubWidget</class>
 <widget class="QWidget" name="SubWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Seat_Search</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;AcadEref&quot;;</string>
   </property>
   <property name="text">
    <string>需要座位数量：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>1-4个</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>550</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>20</y>
     <width>451</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string>Seat_Search</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>420</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>获取位置</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;AcadEref&quot;;</string>
   </property>
   <property name="text">
    <string>当前位置：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>320</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请点击获取位置</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>501</width>
     <height>701</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/ziyuan/bj3.png);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>label_2</zorder>
  <zorder>lineEdit</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>label</zorder>
  <zorder>pushButton</zorder>
  <zorder>label_3</zorder>
  <zorder>lineEdit_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
